### PR TITLE
2021-12-21-Exmo

### PR DESCRIPTION
**Pull request by JediFaust**
Exmo hack article with grammar issues

### CODE REVIEW

**Comment by JediFaust** [comment]
@alxarno grammar checker execution drops this error:
`grammar-check: error: unrecognized arguments: qualitycheck`

**Comment by JediFaust** [comment]
@evgenydmitriev I suggest leaving a separate command for the grammar checker since there were a lot of cases where requested changes related to fact-checking or formatting were addressed, but grammar mistakes have remained.

**Comment by JediFaust** [comment]
Failed claims double checked 👌🏻

**Comment by JediFaust** [comment]
@alxarno I guess dependencies should be installed, since the following error occurred:
`markdown2 is needed to convert markdown to HTML
you can install it with pip:
pip install markdown2`

File: content/attacks/posts/2021-12-21-Exmo.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+entity-types:
+- Exchange
+- Custodian
+attack-types: Wallet Hack
+title: "EXMO Exchange Hot Wallets Compromised: Approximately $10.5 Million Stolen"
+---
+
+## Summary
+
+On December 21, 2021, the attackers gained unauthorized access to the EXMO exchange's hot wallets. Through these security weaknesses, they managed to execute malicious transactions, resulting in the theft of a substantial amount of cryptocurrency.
```
Comment (jhirschkorn) [inline content/attacks/posts/2021-12-21-Exmo.md:13]: is this a different incident from this one: https://info.exmo.com/en/notifications/the-2020-exmo-security-incident-resolved/

I know you double checked the dates but just want to get it straight in my head.

File: content/attacks/posts/2021-12-21-Exmo.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- 1,882.6 BCH
+- 867 ETH
+- 20,651 ETC
+- 476,521 XRP
+- 39,285 ZEC
+- 50,000 USDT
+
+## Timeline
+
+- **December 26, 2017**: Pavel Lerner, EXMO's lead analyst, was kidnapped in Kyiv. A group of unknown masked men snatched Lerner off the street.
```
Comment (jhirschkorn) [inline content/attacks/posts/2021-12-21-Exmo.md:40]: can you add a citation for this one?

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2021-12-21-Exmo.md (hunk 1 above)
Comment (JediFaust) [reply content/attacks/posts/2021-12-21-Exmo.md:13]: My bad, confused years

**Review by jhirschkorn — APPROVED** [review]

**Pull request merged**